evidence/rossbulat/0003-rossbulat-retain.md

```diff
@@ -0,0 +1,107 @@
+- [feat(refactor): Abstract useTimeleft](https://github.com/polkadot-cloud/polkadot-staking-dashboard/pull/2321) 
+- [feat(fix): Render timeleft on end time update](https://github.com/polkadot-cloud/polkadot-staking-dashboard/pull/2320) 
+- [feat: Replace maxElectingVoters const with counterForNominators storage](https://github.com/polkadot-cloud/polkadot-staking-dashboard/pull/2319) 
+- [feat(refactor): Use new Polkicon from @w3ux/polkicon](https://github.com/polkadot-cloud/polkadot-staking-dashboard/pull/2315) 
+- [feat(refactor): Update withProviders from @w3ux/factories](https://github.com/polkadot-cloud/polkadot-staking-dashboard/pull/2314) 
+- [feat(refactor): Use useOnResize, remove lodash.throttle.](https://github.com/polkadot-cloud/polkadot-staking-dashboard/pull/2313) 
+- [feat(refactor): Generalise useSize hook](https://github.com/polkadot-cloud/polkadot-staking-dashboard/pull/2312) 
+- [feat: Wallet Connect support](https://github.com/polkadot-cloud/polkadot-staking-dashboard/pull/2276) 
+
+PRs w3ux Library:
+- [PR for including Frog nodes in the staking dashboard](https://github.com/w3ux/w3ux-library/pull/148) 
+- [Addition of ProStakers.com validator in the community page](https://github.com/w3ux/w3ux-library/pull/147) 
+- [feat(fix): Initial active account formatting correctly](https://github.com/w3ux/w3ux-library/pull/146) 
+- [feat(refactor): Remove ChainSafe Snap](https://github.com/w3ux/w3ux-library/pull/145) 
+- [feat: add rmDecimals to utils, apply to planckToUnit](https://github.com/w3ux/w3ux-library/pull/144) 
+- [feat: Add useTimeleft hook and utils to @w3ux/hooks](https://github.com/w3ux/w3ux-library/pull/143) 
+- [feat(refactor): Misc utils improvements](https://github.com/w3ux/w3ux-library/pull/142) 
+- [feat: add fontSize prop to Polkicon, div -> span](https://github.com/w3ux/w3ux-library/pull/141) 
+- [feat: Rewrite and improve Polkicon - @w3ux/react-polkicon](https://github.com/w3ux/w3ux-library/pull/139) 
+- [feat: Add @w3ux/crypto package](https://github.com/w3ux/w3ux-library/pull/138) 
+- [feat: init factories package with withProviders](https://github.com/w3ux/w3ux-library/pull/137) 
+- [feat: Add useOnResize hook](https://github.com/w3ux/w3ux-library/pull/136) 
+- [feat: Add useSize hook to @w3ux/hooks](https://github.com/w3ux/w3ux-library/pull/134) 
+- [feat(utils):unit fn enhancements + more tests](https://github.com/w3ux/w3ux-library/pull/133) 
+- [feat(refactor): Removes bignumber.js as utils dependency, remove obsolete utils, enhance unit utils](https://github.com/w3ux/w3ux-library/pull/130) 
+- [chore: substrate bindings for AccountId codec](https://github.com/w3ux/w3ux-library/pull/129) 
+- [chore: substrate bindings for AccountId codec](https://github.com/w3ux/w3ux-library/pull/129) 
+
+
+## Voting record
+
+|  Ranks | Activity thresholds | Agreement thresholds | Member's voting activities | Comments |
+|---|---|---|---|---|
+|I  |90%   |N/A   |   |  |
```

Comment:
The voting record is mandatory.
You can write "None" or "No referenda for my rank" to report on your voting activity.